packages/app/components/create.tsx

```diff
@@ -463,65 +480,46 @@ function Create() {
             )}
           />
         </View>
-      </CreateScrollView>
 
-      <View tw="mt-8 w-full px-4">
-        {shouldShowSignMessage ? (
-          <View tw="px-2">
-            {!isSignRequested ? (
+        <View tw="mt-8 w-full px-4">
+          {shouldShowSignMessage ? (
+            <View tw="px-2">
               <Text tw="text-center text-lg dark:text-gray-400">
```

Comment:
Not sure about `text-lg` here: the font is different and the full bold is pretty hard on the eyes. What do you think?

Reply:
Same comment applies to `drop` and `login` too

Reply:
cc - @axeldelafosse